runtime/vm/segment.c

```diff
@@ -639,6 +639,19 @@ allClassesNextDo(J9ClassWalkState* state)
 	return clazzPtr;
 }
 
+J9Class*
+allClassesNextDo(J9ClassWalkState* state)
+{
+	J9Class *clazz = allClassesNextDoInternal(state);
+	while((NULL != clazz)
+         && J9_ARE_ALL_BITS_SET(clazz->classFlags, J9ClassIsFrozen)
+         && J9_ARE_NO_BITS_SET(state->classMask,J9ClassIsFrozen)) {
+		clazz= allClassesNextDoInternal(state);
+	}
+
+	return clazz;
+}
```

Comment:
Formatting, indentation:
```c
J9Class *
allClassesNextDo(J9ClassWalkState *state)
{
	J9Class *clazz = allClassesNextDoInternal(state);
	while ((NULL != clazz)
		&& J9_ARE_ANY_BITS_SET(clazz->classFlags, J9ClassIsFrozen)
		&& J9_ARE_NO_BITS_SET(state->classMask, J9ClassIsFrozen)
	) {
		clazz = allClassesNextDoInternal(state);
	}

	return clazz;
}
```
Note the preference for `J9_ARE_ANY_BITS_SET` instead of `J9_ARE_ALL_BITS_SET` when testing a single bit.

Reply:
There should be a space both sides of `=` in every assignment.